kDrive/UI/View/Files/FileCollectionViewCell.swift

```diff
@@ -192,13 +192,13 @@ class FileCollectionViewCell: UICollectionViewCell, SwipableCell {
     @IBOutlet weak var downloadProgressView: RPCircularProgress?
     @IBOutlet weak var highlightedView: UIView!
 
-    var viewModel: FileViewModel!
+    var viewModel: FileViewModel?
```

Comment:
I'm unable to make it `non nullable` since it is initiated from a xib. This is a quickfix.